src/Applications/LBAF.py

```diff
@@ -149,7 +152,11 @@ def get_conf_file(self, conf_file=os.path.join(project_path, "src", "Application
         """
 
         if os.path.splitext(conf_file)[-1] in [".yml", ".yaml"] and os.path.isfile(conf_file):
-            print(f"{bcolors.OKMSG}Config file {conf_file} FOUND!{bcolors.END}")
+            # Try to open configuration file
+            print(bcolors.HEADER
```

Comment:
Maybe `fstring` here?
```Python
print(f"{bcolors.HEADER}[LBAF]{bcolors.END} Found configuration file {conf_file}")
```

Reply:
Yeah why not. But I don't like the `fstring` syntax. I find it much less clear.

Reply:
Done